resource/graph_node.go

```diff
@@ -189,6 +192,15 @@ func (w *GraphNode) NeedsReconfigure() bool {
 	return !w.markedForRemoval && w.needsReconfigure
 }
 
+// ShouldTryReconfigure returns whether or not we should reconfigure based on how
+// many previous attempts were made.
+func (w *GraphNode) ShouldTryReconfigure() bool {
+	needsReconfigure := w.NeedsReconfigure()
```

Comment:
do we need this call?

Reply:
I don't think we do based on how this new method is used in the resource manager; it will be redundant.